app/src/main/java/com/example/memeitupapp/ui/listed_memes/ListedMemesFragment.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.memeitupapp.ui.listed_memes
```

Comment:
do not use **_** to separate words in packages, use **.**

Reply:
done, thanks